eclipse-collections-api/src/main/java/org/eclipse/collections/api/map/MapIterable.java

```diff
@@ -95,6 +96,25 @@
      */
     void forEachKeyValue(Procedure2<? super K, ? super V> procedure);
 
+    /**
+     * Implements the {@code injectInto} pattern with each <em>key-value</em> pair of the map.
+     * <pre>
+     *     MapIterable&lt;Integer, Integer&gt; map1 = this.newWithKeysValues(3, 3, 2, 2, 1, 1);
```

Comment:
Please replace `this.newWithKeysValues(3, 3, 2, 2, 1, 1)` in the example with `Maps.immutable.with(3, 3, 2, 2, 1, 1)`.